test/integration_test_helper.rb

```diff
@@ -0,0 +1,123 @@
+ENV["RAILS_ENV"] = "test"
```

Comment:
> Since jenkins runs jenkins:unit for functionals/units and jenkins:integration for integration tests, each of the tasks will load a bit faster & with less memory footprint:

I'm not sure it'll affect Jenkins all that much as it runs them in a single rake run, but it'll probably help with single test runs for developers.